db/migrate/20230920210302_create_join_table_hashtags_tweets.rb

```diff
@@ -0,0 +1,8 @@
+class CreateJoinTableHashtagsTweets < ActiveRecord::Migration[7.0]
+  def change
+    create_join_table :hashtags, :tweets do |t|
+      # t.index [:hashtag_id, :tweet_id]
+      # t.index [:tweet_id, :hashtag_id]
+    end
```

Comment:
This table should be the one that has the `hashtag_id` and `tweet_id` created in the previous migration